sdks/wasm/Mono.WebAssembly.DebuggerProxy/DevToolsProxy.cs

```diff
@@ -118,7 +119,10 @@ DevToolsQueue GetQueueForTask (Task task)
 		void Send (WebSocket to, JObject o, CancellationToken token)
 		{
 			var sender = browser == to ? "Send-browser" : "Send-ide";
-			Log ("protocol", $"{sender}: {o}");
+
+			var method = o ["method"]?.ToString ();
+			if (method != "Debugger.scriptParsed" && method != "Runtime.consoleAPICalled")
```

Comment:
Can we remove the protocol log filtering? sadly both of these events are actually critical to the functioning of the proxy and needed to debug problems.

Reply:
We can but the the logging was actually disabled, I suspect because of the sheer volume of these two types of messages.